config/config.go

```diff
@@ -0,0 +1,119 @@
+package config
+
+import (
+	"fmt"
+	"log"
+	"os"
+
+	"github.com/Clever/configure"
+	discovery "github.com/Clever/discovery-go"
+	env "github.com/segmentio/go-env"
+)
+
+var (
+	// things we are likely to change when running the worker normally are flags
+	InputSchemaName string
+	InputTables     string
+	InputBucket     string
+	Truncate        bool
+	Force           bool
+	DataDate        string
+	ConfigFile      string
+	Gzip            bool
+	Delimiter       string
+	TimeGranularity string
+	StreamStart     string
+	StreamEnd       string
+	TargetTimezone  string
+
+	// things which will would strongly suggest launching as a second worker are env vars
+	// also the secrets ... shhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhh
+	Host            string //= os.Getenv("REDSHIFT_HOST")
+	Port            string //= os.Getenv("REDSHIFT_PORT")
+	DbName          string //= env.MustGet("REDSHIFT_DB")
+	User            string //= env.MustGet("REDSHIFT_USER")
+	Pwd             string //= env.MustGet("REDSHIFT_PASSWORD")
+	RedshiftRoleARN string //= env.MustGet("REDSHIFT_ROLE_ARN")
+	VacuumWorker    string //= env.MustGet("VACUUM_WORKER")
+
+	// payloadForSignalFx holds a subset of the job payload that
+	// we want to alert on as a dimension in SignalFx.
+	// This is necessary because we would like to selectively group
+	// on job parameters - schema but not date, for instance, since
+	// logging the date would overwhelm SignalFx
+	PayloadForSignalFx string
+
+	GearmanAdminURL string
+)
+
+type cliConfig struct {
+	InputSchemaName string `config:"schema"`
+	InputTables     string `config:"tables"`
+	InputBucket     string `config:"bucket"`
+	Truncate        bool   `config:"truncate"`
+	Force           bool   `config:"force"`
+	DataDate        string `config:"date"`
+	ConfigFile      string `config:"config"`
+	Gzip            bool   `config:"gzip"`
+	Delimiter       string `config:"delimiter"`
+	TimeGranularity string `config:"granularity"`
+	StreamStart     string `config:"streamStart"`
+	StreamEnd       string `config:"streamEnd"`
+	TargetTimezone  string `config:"timezone"`
+}
+
+func Parse() {
+	argConfig := cliConfig{
+		InputSchemaName: "mongo",
+		InputBucket:     "metrics",
+		Truncate:        false,
+		Gzip:            true, // TODO: currently this can not be set to false
```

Comment:
@ahsojar this is an issue currently tracked at https://github.com/Clever/configure/pull/9